homeassistant/components/triggercmd/config_flow.py

```diff
@@ -0,0 +1,62 @@
+"""Config flow for TRIGGERcmd integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import jwt
+import voluptuous as vol
+
+from homeassistant import config_entries, exceptions
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema({("token"): str})
+
+
+async def validate_input(hass: HomeAssistant, data: dict) -> dict[str, Any]:
+    """Validate the user input allows us to connect.
+
+    Data has the keys from DATA_SCHEMA with values provided by the user.
+    """
+    if len(data["token"]) < 100:
+        raise InvalidToken
+
+    tokenData = jwt.decode(data["token"], options={"verify_signature": False})
```

Comment:
Can we somehow validate the token by doing a request?
Currently we are only checking if it's any jwt token, but not if the token is a TriggerCmd jwt token

Reply:
I added an API request to validate the token.